requirements.txt

```diff
@@ -1 +1,3 @@
 -e .
+aioresponses
```

Comment:
I used to list requirements in setup.py file. I think that requirements for tests should not be listed in install_requires but let's use tests_require section for that. It is deprecated but easier to use that extras_require that replaced it

Reply:
Dependencies are not installed with `test_require`. Maybe we should use the `requirements-test.txt` file?